api/organisations/models.py

```diff
@@ -95,6 +96,11 @@ def num_seats(self):
     def has_subscription(self) -> bool:
         return hasattr(self, "subscription") and bool(self.subscription.subscription_id)
 
+    def has_subscription_information_cache(self) -> bool:
+        return hasattr(self, "subscription_information_cache") and bool(
+            self.subscription_information_cache.id
```

Comment:
I'm not certain if this will work, but it would be preferable, if possible, to do this:

```suggestion
            self.subscription_information_cache_id
```

Reply:
this does not work:
`subscription_information_cache_id`
but this works:
`subscription_information_cache`